### PR TITLE
scan: implement and wireup redelegate/withdraw reward msgs

### LINKED ISSUE #1426: Implement UI & Wire up on Redelegate
- implement UI as a design on #1423
- implement creating redelegate msg like request
- show error properly if balance is the insufficient or invalid input field
- testing with mnemonic and ledger

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1426, #1428

### CODE REVIEW

File: scan/src/components/modal/submitTx/RedelegateMsg.re [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            srcValidatorValue,
+            dstValidatorValue,
+            {amount: amountValue |> string_of_int, denom: "uband"},
+          ),
+        |]);
+      };
+      setMsgsOpt(_ => msgsOpt);
+      None;
+    },
+    (dstValidator, amount),
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/RedelegateMsg.re]: Let's use `React.useEffect3` for adding `srcValidator` as dependencies.

File: scan/src/components/modal/submitTx/WithdrawRewardMsg.re [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  React.useEffect2(
+    _ => {
+      let msgsOpt = {
+        let%Opt validatorValue = validator.value;
+        Some([|TxCreator.WithdrawReward(validatorValue)|]);
+      };
+      setMsgsOpt(_ => msgsOpt);
+      None;
+    },
+    (validator, 0),
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/WithdrawRewardMsg.re]: Let's use `React.useEffect1` for one dependency.

**Review by evilpeach — CHANGES_REQUESTED** [review]
The space between the button and the last field is too close. Let's decrease `padding-bottom` of modal's container.

File: scan/src/components/modal/submitTx/WithdrawRewardMsg.re [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  React.useEffect1(
+    _ => {
+      let msgsOpt = {
+        let%Opt validatorValue = validator.value;
+        Some([|TxCreator.WithdrawReward(validatorValue)|]);
+      };
+      setMsgsOpt(_ => msgsOpt);
+      None;
+    },
+    validator |> Belt.Array.make(1, _),
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/WithdrawRewardMsg.re]: Why don't use `[| validator |]`

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by prin-r — APPROVED** [review]

**Pull request merged**